bitnami/etcd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: etcd
-version: 4.6.4
+version: 4.6.5
```

Comment:
I would add a minor version since we are adding a new value

Reply:
Agreed, done!